### PR TITLE
Extremely Ultimate Capacitor

### PR DESCRIPTION
**Pull request by GDCloudstrike**
Adds the Extremely Ultimate Capacitor, also known as the UEV capacitor.
It has the same EU capacity as the Ultimate Capacitor, but has 100x wireless power throughput capabilities (1 Trillion eu/t).
For that reason it also has 100x passive loss (100M eu/t). 
The power requirements in endgame are reaching trillions of eu/t which wireless power can't reasonable handle with the 10 beu/t transfer limit of UHV capacitors.
The UEV capacitor aims to resolve that issue and allow players to use their power in the wireless network to the fullest.

Its recipe follows the trend of the lapotronic batteries, but with higher tier materials and parts.
image

### CODE REVIEW

File: src/main/java/common/Recipes.java [hunk 1]
```diff
@@ -924,6 +924,53 @@ private static void registerRecipes_LSC() {
...
+                },
+                new FluidStack[] {
+                    new FluidStack(solderUEV, 9216),
+                    Materials.Quantium.getMolten(18432),
+                    Materials.Naquadria.getMolten(18432),
+                    Materials.SuperCoolant.getFluid(64000)
+                },
+                new ItemStack(Blocks.lscLapotronicEnergyUnit, 1, 8),
+                250 * 20,
+                6_400_000);
```
Comment (BlueWeabo) [inline src/main/java/common/Recipes.java]: use TierEU.RECIPE_UEV/UIV

File: src/main/java/common/Recipes.java [hunk 2]
```diff
@@ -924,6 +924,53 @@ private static void registerRecipes_LSC() {
...
+                new ItemStack[] {
+                    ItemList.ZPM4.get(1),
+                    GT_OreDictUnificator.get(OrePrefixes.frameGt, Materials.Infinity, 4),
+                    GT_OreDictUnificator.get(OrePrefixes.screw, Materials.Infinity, 24),
+                    GT_Utility.getIntegratedCircuit(6)
+                },
+                null,
+                new ItemStack(Blocks.lscLapotronicEnergyUnit, 1, 8),
+                640 * 20,
+                BW_Util.getMachineVoltageFromTier(9));
```
Comment (BlueWeabo) [inline src/main/java/common/Recipes.java]: TierEU.RECIPE_UHV

File: src/main/java/common/Recipes.java [hunk 3]
```diff
@@ -961,6 +1008,12 @@ private static void registerRecipes_LSC() {
                 GT_OreDictUnificator.get(OrePrefixes.screw, Materials.CosmicNeutronium, 24),
                 1200,
                 32);
+        GT_Values.RA.addUnboxingRecipe(
+                new ItemStack(Blocks.lscLapotronicEnergyUnit, 1, 8),
+                ItemList.ZPM4.get(1L),
+                GT_OreDictUnificator.get(OrePrefixes.screw, Materials.Infinity, 24),
+                1200,
+                32);
```
Comment (BlueWeabo) [inline src/main/java/common/Recipes.java]: why exactly 32 eu/t :concern: fix the others too with TierEU.RECIPE_LV

**Review by BlueWeabo — CHANGES_REQUESTED** [review]

**Comment by BlueWeabo** [comment]
Apply spotless and i will approve

**Comment by GDCloudstrike** [comment]
> Apply spotless and i will approve

That's the thing, it doesn't work 😭

**Comment by BlueWeabo** [comment]
:concern:

**Comment by BlueWeabo** [comment]
well yeah i just tried it. and nope. uhhh

**Review by BlueWeabo — APPROVED** [review]
welp. its smt internal on rfg. not my problem XD

**Pull request merged**